src/components/tx/confirmation-views/SettingsChange/index.ts

```diff
@@ -0,0 +1,3 @@
+import SettingsChange from './SettingsChange'
+
+export default SettingsChange
```

Comment:
👍 I usually try to follow module pattern, where you have this kind of folder structure `[component]/index` and , `[component]/[component]` in order to be easy to find when searching for the file name. But your request makes sense, because I can see having the component direct in the index is the standard of the project 👍